docs/source/getting_started/using_entitysets.ipynb

```diff
@@ -0,0 +1,337 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Representing Data with EntitySets\n",
+    "\n",
+    "An ``EntitySet`` is a collection of dataframes and the relationships between them. They are useful for preparing raw, structured datasets for feature engineering. While many functions in Featuretools  take ``dataframes`` and ``relationships`` as separate arguments, it is recommended to create an ``EntitySet``, so you can more easily manipulate your data as needed.\n",
+    "\n",
+    "## The Raw Data\n",
+    "\n",
+    "Below we have two tables of data (represented as Pandas DataFrames) related to customer transactions. The first is a merge of transactions, sessions, and customers so that the result looks like something you might see in a log file:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import featuretools as ft\n",
+    "data = ft.demo.load_mock_customer()\n",
+    "transactions_df = data[\"transactions\"].merge(data[\"sessions\"]).merge(data[\"customers\"])\n",
+    "\n",
+    "transactions_df.sample(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And the second dataframe is a list of products involved in those transactions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "products_df = data[\"products\"]\n",
+    "products_df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating an EntitySet\n",
+    "\n",
+    "First, we initialize an ``EntitySet``. If you'd like to give it a name, you can optionally provide an ``id`` to the constructor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "es = ft.EntitySet(id=\"customer_data\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding dataframes\n",
+    "\n",
+    "To get started, we add the transactions dataframe to the entity set. In the call to ``add_dataframe``, we specify three important parameters:\n",
```

Comment:
`entity set` -> `EntitySet`

Also maybe check that this is consistent throughout the document.

Reply:
updated in dfe54852